packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -333,12 +333,17 @@ class GlobalObjectKey<T extends State<StatefulWidget>> extends GlobalKey<T> {
   }
 }
 
-/// This class is a work-around for the "is" operator not accepting a variable value as its right operand
+/// This class is a work-around for the "is" operator not accepting a variable
+/// value as its right operand, and for the fact that generics aren't valid type
+/// literals (e.g. "Type type = MyClass<int>;" is invalid).
```

Comment:
backticks

Reply:
add `//` links to the relevant bugs, maybe as TODOs to remove this class when these issues are resolved (https://github.com/dart-lang/sdk/issues/27680 and https://github.com/dart-lang/sdk/issues/33297)